README.md

```diff
@@ -1,272 +1,144 @@
-
-- The integrate function lets you choose a specific Monte Carlo (MC) algorithm by using the `solver` keyword argument. The example given employs the Vegas algorithm with `:vegas`. Additionally, this package provides two Markov-chain Monte Carlo (MCMC) algorithms for numerical integration: `:vegasmc` and `:mcmc`. Comparing these MCMC algorithms, `:vegasmc` offers better accuracy than `:mcmc` while keeping the same robustness. Although `:vegas` is generally slightly more accurate than `:vegasmc`, it is less robust. Considering the trade-off between accuracy and robustness, integrate defaults to using `:vegasmc`. For further information, consult the [Algorithm](#Algorithm) section.
-
-- When defining your own integrand evaluation function, you need to provide two arguments: `(x, c)`:
-  * `x` represents the integration variable, which by default falls within the range [0, 1). It should be considered as a pool of infinitely many random variables that follows the same distribution. To access the i-th random variable, use x[i]. For a better understanding, refer to Example 2 and the [Variables](#Variables) section.
-  * `c` is a struct that holds the Monte Carlo (MC) configuration. This contains additional information that might be necessary for evaluating the integrand. For a practical example, see Example 5.
-
-- For complex-valued integral, say with the type `ComplexF64`, you need to call `integrate(..., dtype = ComplexF64)` to specify the integrand data type. The error  of the real part and the imaginary part will be estimated independently.   
-
-- You can suppress the output information by setting `print=-1`. If you want to see more information after the calculation, simply call `report(res)`. If you want to check the MC configuration, call `report(res.config)`.
-
-## Example 2. Multi-dimensional integral: Symmetric Variables
-
-In `MCIntegration.jl`, a variable is represented as a pool of random numbers drawn from the same distribution. For instance, you can explicitly initialize a set of variables in the range [0, 1) as follows:
-```julia
-julia> x=Continuous(0.0, 1.0) #Create a pool of continuous variables. 
-Adaptive continuous variable in the domain [0.0, 1.0). Max variable number = 16. Learning rate = 2.0.
+pkg> add MCIntegration
 ```
-This approach simplifies the evaluation of high-dimensional integrals involving multiple symmetric variables. For example, to calculate the area of a quarter unit circle (π/4 = 0.785398...):
+## Quick Start
+MCIntegration.jl simplifies complex integral calculations. Here are two examples to get you started.
+
+To estimate the integral $\int_0^1 \frac{\log(x)}{\sqrt{x}} dx = 4$, you can use:
 ```julia
-julia> res = integrate((x, c)->(x[1]^2+x[2]^2<1.0); var = x, dof = 2) 
-Integral 1 = 0.7860119307731648 ± 0.002323473435947719   (chi2/dof = 2.14)
+julia> f(x, _) = log(x[1]) / sqrt(x[1])   # Define your integrand where 'x' represents the random variable in the integral
```

Comment:
Use f(x, c) to be consistent with the following "an integrand function `f(x, c)`"